### ISSUE TITLE
The "Open link in sidebar" context menu option does not work if the sidebar's Homepage UI is open

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/tab-split-2#74]
[Affected versions]:
- Firefox 58.0.2 and up
- Tab Split 2 0.2.0

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have a clean Firefox profile with the latest Tab Split 2 v0.2.0 installed.
- Have the Tab Split sidebar open.

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites and navigate to a website.
2. Right click on a hyperlinked element.
3. Choose the "Open link to sidebar" option and observe the sidebar.

[Expected result]:
- The link is loaded in the sidebar.

[Actual result]:
- Nothing happens.

[Notes]:
- This issue is also reproducible for the "Open in sidebar" option from the tab strip context menu.
- Here is a screen recording with this issue:
![tab split 2 - open link](https://user-images.githubusercontent.com/31244235/37213327-645781be-23ba-11e8-84b0-5e605e15aa3a.gif)

**Comment by ianb** [comment]
@johngruen found something like this too (#62 I think) but I wasn't able to reproduce it. But I'm also not able to reproduce this. I'm doing:

1. Run `npm start`
2. Open a site (I opened https://developer.mozilla.org/en-US/)
3. Right-click on a link (e.g., Technologies)
4. Select Open link in sidebar

... and it all works like I'd expect. Am I missing something? Does something show up in the Browser Console?

**Comment by Softvision-CristinaBadescu** [comment]
I've made a mistake when I first reported the issue, it was never reproducible on an unbranded Release or Beta build.

Moreover it looks like it is no longer reproducible on latest Nightly build 60.0a1 (2018-03-12), using the latest Tab Split 2 build, on Windows 10 x64, Mac 10.13.3 and Arch Linux 4.12. 

I've investigated this change and have found that [bug 1398713](https://goo.gl/zAwV7D), has "fixed" this issue (unlinke in some other cases).

Considering this I will close this issue as it is no longer reproducible.
